dotCMS/src/curl-test/Apps Resource Test.postman_collection.json

```diff
@@ -808,6 +808,162 @@
 			],
 			"protocolProfileBehavior": {}
 		},
+		{
+			"name": "import-export",
```

Comment:
Please add a description field on each postman test to know the scenario each test is considering

Reply:
done